src/components/Card.js

```diff
@@ -23,9 +49,16 @@ export default class Card {
     this.#cardImage = this.#cardElement.querySelector(".card__image");
     this.#cardLikeButton =
       this.#cardElement.querySelector(".card__button-like");
+    this.setIsLiked(this.#likesArray);
+
+    this.toggleActiveLike();
+    this.#cardLikeAmount =
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**